cli/impl/src/main/assembly/helidon.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
This file needs to omit the `.sh` extension ; otherwise users will have to type `helidon.sh` explicitly. We want them to type `helidon`.